pkg/scheduler/framework/plugins/nodeaffinity/node_affinity_test.go

```diff
@@ -42,14 +42,15 @@ func TestNodeAffinity(t *testing.T) {
 		wantPreFilterStatus *framework.Status
 		wantPreFilterResult *framework.PreFilterResult
 		args                config.NodeAffinityArgs
-		disablePreFilter    bool
+		runPreFilter        bool
```

Comment:
Why? `disablePreFilter` default to false matches the current behavior.

Reply:
I think it's because this comment https://github.com/kubernetes/kubernetes/pull/115655#discussion_r1128556218

Reply:
`disableX=false` and `runX=true` are equivalent, but `runX=true` is significantly easier to read.

Reply:
`runX=true` is easier to read, `disableX=false` matches the default behavior , not a big matter, so
/lgtm